packages/cardpay-sdk/sdk/reward-manager/base.ts

```diff
@@ -1055,9 +1070,28 @@ The owner of reward safe ${safeAddress} is ${rewardSafeOwner}, but the signer is
   async getGovernanceAdmin(): Promise<string> {
     return await (await this.getRewardManager()).methods.governanceAdmin().call();
   }
-  async getRewardRule(rewardProgramId: string): Promise<string> {
+  async getBlob(rewardProgramId: string): Promise<string> {
     return await (await this.getRewardManager()).methods.rule(rewardProgramId).call();
   }
+  async getRuleDid(rewardProgramId: string): Promise<string> {
+    const blob = await this.getBlob(rewardProgramId);
+    return Buffer.from(blob.replace('0x', ''), 'hex').toString('utf-8');
+  }
+
+  async getRuleJson(rewardProgramId: string): Promise<RuleJson> {
+    const did = await this.getRuleDid(rewardProgramId);
+    const content = await resolveDoc(did);
+    return content;
+  }
+
+  getProgramExplainer(rule: RuleJson) {
+    return get(rule, 'explanation.en.program', undefined);
+  }
+
+  getClaimExplainer(rule: RuleJson, explanationId: string) {
+    return get(rule, 'explanation.en.claim.' + explanationId, undefined);
```

Comment:
We can default to en, but if we add this as at least an optional argument we don't have to modify the sdk in the future to support multiple locales.